### PR TITLE
Fixes #628 by adding link to footer

### LINKED ISSUE #628: Link to project GitHub on the site
We don't currently.

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Before:
image

After:
image

### CODE REVIEW

File: app/views/application/_footer.haml [hunk 1]
```diff
@@ -11,6 +11,9 @@
       %li
         Read our blog
         = link_to "openaustraliafoundation.org.au/blog", "http://www.openaustraliafoundation.org.au/blog"
+      %li
+        Fork this project
```
Comment (henare) [inline app/views/application/_footer.haml]: "Fork" is a bit jargony for this project - the general audience aren't programmers so wouldn't understand what it meant. Can we improve this wording?

Comment (CloCkWeRX) [inline app/views/application/_footer.haml]: While that's true, its the github link - wouldn't the intended users be developers?

How does just "Github" or "Us on Github" work for you?

Comment (equivalentideas) [inline app/views/application/_footer.haml]: Could also be 'source code' like on morph.io . or "Source code on GitHub" ?

For people who aren't devs who want to see what's 'behind the scenes' “source code” is more likely a familiar term or something they can work out, I think.

**Comment by equivalentideas** [comment]
Nice work with this @CloCkWeRX Could you please add a before and after screenshot so we can see the impact of adding another link there to the layout?

**Comment by CloCkWeRX** [comment]
Updated

File: app/views/application/_footer.haml (hunk 1 above)
Comment (henare) [inline app/views/application/_footer.haml]: morph.io also has "GitHub".

**Comment by equivalentideas** [comment]
Great thanks @CloCkWeRX .

It looks like the right is becoming more about OAF and the left about PlanningAlerts. Think we should move the blog link to the right to be with the conceptually similar stuff and rebalance the layout?

**Comment by CloCkWeRX** [comment]
Hard to get the vertical heights right without changing a bunch more of the headings; or putting in some kind of grid layout.

image

**Comment by equivalentideas** [comment]
@CloCkWeRX I think that's more balanced than it currently is ;-)

I'm happy with that, just make sure it's all good on small screens too.

**Comment by CloCkWeRX** [comment]
image

**Comment by equivalentideas** [comment]
Nice work @CloCkWeRX I've merged this locally.

I removed the `<br />` added and recreated the layout in css in 3cb78ff .

**Comment by equivalentideas** [comment]
Deployed 🚢 Thanks @CloCkWeRX !

**Pull request merged**